lib/docker.ts

```diff
@@ -186,14 +226,27 @@ class Docker {
         }
       }
       if (!exclude) {
-        for (const g of globs) {
-          if (minimatch(filepath, g)) {
-            res.push(filepath);
+        let foundInManifestGlobs;
+
+        if (globs.manifestGlobs) {
+          for (const g of globs.manifestGlobs) {
+            if (minimatch(filepath, g)) {
+              foundInManifestGlobs = true;
```

Comment:
don't you want to `break` on that case? also in the loop below

Reply:
yes, fixed